packages/react/src/components/Popup/Popup.tsx

```diff
@@ -120,6 +133,35 @@ function useRestoreFocus(props: IPopupProps, root: React.RefObject<HTMLDivElemen
   );
 }
 
+function ariaHidden(nodesToHide: HTMLElement[], show: boolean) {
+  if (show) {
+    nodesToHide.forEach(child => child.setAttribute('aria-hidden', 'true'));
+  } else {
+    nodesToHide.forEach(child => child.removeAttribute('aria-hidden'));
+  }
+}
+
+function useGetNodesToHide(props: IPopupProps): HTMLElement[] | undefined {
+  const [nodesToHide, setNodesToHide] = React.useState<HTMLElement[]>();
+  const isModalOrPanel = props['aria-modal'];
+
+  React.useEffect(() => {
+    if (isModalOrPanel) {
+      const blackListTagNames = ['TEMPLATE', 'SCRIPT', 'STYLE'];
+      const bodyChildren = getChildren(getDocument()!.body);
+
+      const filteredBodyChildren = bodyChildren
+        .slice(0, bodyChildren.length - 1)
```

Comment:
I think this is because the `Layer` element where the `Popup` appears is always guaranteed, in our implementation, to be the last child from that list, so it is just removing it by default.

Reply:
Yup exactly as @khmakoto said!